libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/ui/SalesforceDroidGapActivity.kt

```diff
@@ -581,21 +584,26 @@ open class SalesforceDroidGapActivity : CordovaActivity(), SalesforceActivityInt
     }
 
     private fun doAuthConfig() {
-        val loginServer = SalesforceHybridSDKManager
-            .getInstance()
-            .loginServerManager
-            .selectedLoginServer
-            ?.url
-            ?.trim { it <= ' ' } ?: return
-
-        if (loginServer == PRODUCTION_LOGIN_URL || loginServer == SANDBOX_LOGIN_URL || !isHttpsUrl(loginServer) || parse(loginServer) == null
-        ) {
-            return
-        }
-
-        authConfig = getMyDomainAuthConfig(loginServer)
+        CoroutineScope(Default).launch {
```

Comment:
This just catches the call to `getMyDomainAuthConfig` here up with the concurrency and logging code that's in the `SalesforceSdkManager` version.  It _is_ redundant, but does avoid any more intrusive change to the pending 12.0 release.